packages/components/src/internal/components/entities/EntityLineageEditModal.tsx

```diff
@@ -200,13 +200,17 @@ export const EntityLineageEditModal: FC<Props> = memo(props => {
 
             <Modal.Footer>
                 {onCancel && (
-                    <Button className="pull-left" onClick={onCancel}>
+                    <Button
+                        className="pull-left test-loc-cancel-button"
```

Comment:
I'm curious why these locators are needed.  Often it's sufficient to find a button with text. Is there more than one "Cancel" button available when testing this?  (I'm not generally a fan of these do-nothing test locator classes.  They are apparent enough with the naming convention, of course, just kind of cluttering.)

Reply:
I was following what had been done for the locators in the `BatchUpdateSamplesDialog.java` class. For the cancel button, there is only one so it would be easy enough to drop this locator and just use the button text. For the "success/save" button, it could be either "Update Sources" or "Update Parents", but I guess we could just do a locator that checks for a button that starts with or contains "Update".

Reply:
This is definitely my preference.